src/components/Utilities/UtilitiesTable.tsx

```diff
@@ -57,9 +66,11 @@ function UtilitiesTable(props: Props) {
       const lrIndex = lastRowSelected?.current?.index ?? row.index;
       const fromIndex = Math.min(lrIndex, row.index);
       const toIndex = Math.max(lrIndex, row.index);
+      const isSelected = lastRowSelected.current?.getIsSelected() ?? true;
```

Comment:
This is not a complaint to you but to myself. I don't know why I had that inside the loop 🤦